tests/autoria/test_average.py

```diff
@@ -0,0 +1,51 @@
+import requests_mock
+import json
+
+from autoria.api import RiaAverageCarPrice
+
+
+class TestAverage:
+    """Tests for autoria average car price calculation."""
+
+    def test_average_minimal(self, ria_categories, ria_marks, ria_states,
```

Comment:
Parameters passed here are `pytest` [fixtures](https://docs.pytest.org/en/latest/fixture.html) which are automatically loaded from the conftest.py.